src/Responses/FineTuning/RetrieveJobResponseError.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenAI\Responses\FineTuning;
+
+use OpenAI\Contracts\ResponseContract;
+use OpenAI\Responses\Concerns\ArrayAccessible;
+
+/**
+ * @implements ResponseContract<array{code: string, param: string, message: string}>
+ */
+final class RetrieveJobResponseError implements ResponseContract
+{
+    /**
+     * @use ArrayAccessible<array{code: string, param: string, message: string}>
+     */
+    use ArrayAccessible;
+
+    private function __construct(
+        public readonly string $code,
+        public readonly string $param,
```

Comment:
```suggestion
        public readonly ?string $param,
```

According to the docs this is nullable: https://platform.openai.com/docs/api-reference/fine-tuning/object

Reply:
thanks... updated